descholar-ceo/find-flight-info
# User should be able to signup

#### What does this PR ​ do​ ?
It allows user to signup

#### Description of Task to be completed?
* Change the folder structure, from default create-react-app to desired working folder structure
* Create SignupPage component
* Create handlers folder
* Put functions in handlers folder to helps you manage signup fields

#### How should this be manually tested?
* Clone this repository by `git clone https://github.com/descholar-ceo/find-flight-info`
* Type these commands : 
    *  `cd find-flight-info` to go in the root directory of the project
    * `npm install` or `yarn add` to install all the packages
    * `npm start` or `yarn start`
* Then go in the browser, you will see a default opened page is signup


#### Any background context you want to provide?
* You have to install [nodejs](https://nodejs.org/en/) version 6 or higher prior to test this application

#### What are the relevant pivotal tracker stories?

#### Screenshots (​ if​ appropriate)
![01 signupPage](https://user-images.githubusercontent.com/56886064/74930257-db4b9000-53e5-11ea-89be-fbc929f05940.png)

#### Questions:
No Question
